### ISSUE TITLE
Opening files shows an empty buffer

### ISSUE THREAD

**Issue by rafamadriz** [kyazdani42/nvim-tree.lua#507]
I'm having the following behavior of opening files and it only shows an empty buffer.

https://user-images.githubusercontent.com/67771985/125175883-c3350600-e1be-11eb-9946-6afd64371d40.mp4

I've tried using the defaults without any configuration but doesn't work, still the same behavior. I found that reverting to commit [`fd7f60e242205ea9efc9649101c81a07d5f458bb`](https://github.com/kyazdani42/nvim-tree.lua/commit/fd7f60e242205ea9efc9649101c81a07d5f458bb) fixes the issue for me, trying any other commit after that gives the bug showed in the video. So right now I have this in my config with packer to lock NvimTree at that commit.

```lua
    use {
        "kyazdani42/nvim-tree.lua",
        commit = "fd7f60e242205ea9efc9649101c81a07d5f458bb",
        wants = "nvim-web-devicons",
        cmd = { "NvimTreeToggle", "NvimTreeFindFile" },
        config = function()
            require "plugins.tree"
        end,
    }
``` 

It seems like I'm not the only one, I was looking at LunarVim and they have the same, you can see [here](https://github.com/ChristianChiarulli/LunarVim/blob/5c7600322848fa1a55482e2b8b45c7f97cf67a85/lua/plugins.lua#L92), but to be honest I don't know the reasons of why they locked NvimTree at that commit. 

I tried to reproduce this with a minimal `init.lua` and I couldn't, I really don't know what's going on, I really doubt it's my nvimtree configuration, it was all working a few days ago, and I also tried using just the defaults without requiring the file for nvimtree config but that didn't work either. I don't know what the commit after [`fd7f60e242205ea9efc9649101c81a07d5f458bb`](https://github.com/kyazdani42/nvim-tree.lua/commit/fd7f60e242205ea9efc9649101c81a07d5f458bb) did but it doesn't work for me.

**Comment by Johan-Palacios** [comment]
I have the same Issue

**Comment by Yagua** [comment]
I confirm that I also have the same issue

**Comment by dadyarri** [comment]
I have the same

**Comment by sujaybokil** [comment]
I can confirm I have the same issue too, especially while opening recent files from vim-startify

**Comment by nacknime-official** [comment]
Yes, the same.

**Comment by kyazdani42** [comment]
hello everyone. First of all, i'd like to say that `it happens to me too` will not help me to find where this bug comes from.
secondly, i've had this last week and i fixed in in 102c4c2.
lastly, the commit you've pointed out does not affect the rendering process or the file opening process in any way.

Please do update nvim-tree version or provide a minimal sample config, a neovim version and an nvim-tree version from which the bug arise so i can reproduce this.

Thanks.

**Comment by kyazdani42** [comment]
closed in  #524
